dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyPanelFinancials.vue

```diff
@@ -0,0 +1,168 @@
+            />
+          </div>
+          <div
+            class="col-6"
+            v-if="
+              dataSet.creditInstitutionKpis.tradingPortfolio ||
+              !dataSet.creditInstitutionKpis.tradingPortfolioAndInterbankLoans
+            "
+          >
+            <TaxoCard
+              name="tradingPortfolio"
+              title="Trading portfolio"
+              :percent="dataSet.creditInstitutionKpis.tradingPortfolio"
+            />
+          </div>
+          <div
+            class="col-6"
+            v-if="
+              dataSet.creditInstitutionKpis.interbankLoans ||
+              !dataSet.creditInstitutionKpis.tradingPortfolioAndInterbankLoans
+            "
+          >
+            <TaxoCard
+              name="onDemandInterbankLoans"
+              title="On demand interbank loans"
+              :percent="dataSet.creditInstitutionKpis.interbankLoans"
+            />
+          </div>
+        </template>
+        <template v-if="fsType === 'InsuranceOrReinsurance'">
+          <div class="col-12 text-left pb-0">
+            <span class="font-medium text-xl">Insurance and Reinsurance KPIs</span>
+            <span class="pl-2 font-italic text-gray-100">In percentage of the total assets</span>
+          </div>
+          <div class="col-6">
+            <TaxoCard
+              name="taxonomyEligibleNonLifeInsuranceActivities"
+              title="Taxonomy-eligible non-life insurance economic activities"
+              :percent="dataSet.insuranceKpis.taxonomyEligibleNonLifeInsuranceActivities"
+            />
+          </div>
+        </template>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import { ApiClientProvider } from "@/services/ApiClients";
+import TaxoCard from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxoCard";
+import TaxoInfoCard from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxoInfoCard";
+
+export default {
+  name: "EuTaxonomyPanelFinancials",
+  components: { TaxoCard, TaxoInfoCard },
+  data() {
+    return {
+      dataSet: null,
+    };
+  },
+  props: {
+    dataID: String,
+  },
+  mounted() {
+    this.getCompanyEUDataset();
+  },
+  watch: {
+    dataID() {
+      this.getCompanyEUDataset();
+    },
+  },
+  inject: ["getKeycloakPromise"],
+  methods: {
+    async getCompanyEUDataset() {
+      try {
+        const euTaxonomyDataForFinancialsControllerApi = await new ApiClientProvider(
+          this.getKeycloakPromise()
+        ).getEuTaxonomyDataForFinancialsControllerApi();
+        const companyAssociatedData = await euTaxonomyDataForFinancialsControllerApi.getCompanyAssociatedData1(
+          this.dataID
+        );
+        this.dataSet = companyAssociatedData.data.data;
+      } catch (error) {
+        console.error(error);
+      }
+    },
+    getSectionHeading(type) {
+      const mapping = {
+        CreditInstitution: "Credit Institutions",
```

Comment:
Sollte das hier auch irgendwo anders hin exportiert werden. Das mapping wird nur hier gebraucht, passt aber nicht zu humaniseString weil hier in Plural?

Reply:
Solange das mapping nur an einer Stelle auftaucht ist das für mich in Ordnung. Sobald wir es ein zweites mal brauchen würde ich mir dann Gedanken darüber machen.